src/taipy/core/submission/submission.py

```diff
@@ -44,13 +44,15 @@ class Submission(_Entity, _Labeled):
     def __init__(
         self,
         entity_id: str,
+        entity_type: str,
```

Comment:
💬 It means that we can pass any str (i.e. not an enum). 
What should we expect if we pass a string that does not correspond to a real entity?

Reply:
hmmm good point, as this is only an entity for internal usage, I hope this won't be an issue 😅